T-Head/Xuantie-C910-2023-02-26/c910_isa_checked.yaml

```diff
@@ -0,0 +1,4865 @@
+                    wlrl:
+                      - 0:1
+            vstie:
+                implemented: true
+                description: VS-level Timer Interrupt enable.
+                shadow:
+                shadow_type: rw
+                msb: 6
+                lsb: 6
+                type:
+                    warl:
+                        dependency_fields: []
+                        legal:
+                          - vstie[0] in [0x0, 0x1]
+                        wr_illegal:
+                          - unchanged
+            mtie:
+                implemented: true
+                description: Machine Timer Interrupt enable.
+                shadow:
+                shadow_type: rw
+                msb: 7
+                lsb: 7
+                type:
+                    wlrl:
+                      - 0:1
+            ueie:
+                implemented: false
+                description: User External Interrupt enable.
+                shadow:
+                shadow_type: rw
+                msb: 8
+                lsb: 8
+            seie:
+                implemented: true
+                description: Supervisor External Interrupt enable.
+                shadow:
+                shadow_type: rw
+                msb: 9
+                lsb: 9
+                type:
+                    wlrl:
+                      - 0:1
+            vseie:
+                implemented: true
+                description: VS-level External Interrupt enable.
+                shadow:
+                shadow_type: rw
+                msb: 10
+                lsb: 10
+                type:
+                    warl:
+                        dependency_fields: []
+                        legal:
+                          - vseie[0] in [0x0, 0x1]
+                        wr_illegal:
+                          - unchanged
+            meie:
+                implemented: true
+                description: Machine External Interrupt enable.
+                shadow:
+                shadow_type: rw
+                msb: 11
+                lsb: 11
+                type:
+                    wlrl:
+                      - 0:1
+            sgeie:
+                implemented: false
+                description: HS-level External Interrupt enable.
+                shadow:
+                shadow_type: rw
+                msb: 12
+                lsb: 12
+        description: The mie register is an MXLEN-bit read/write register containing
+            interrupt enable bits.
+        address: 772
+        priv_mode: M
+        reset-val: 0
+    mscratch:
+        rv32:
+            accessible: false
+        rv64:
+            accessible: true
+            fields: []
+            shadow:
+            shadow_type: rw
+            msb: 63
+            lsb: 0
+            type:
+                warl:
+                    dependency_fields: []
+                    legal:
+                      - mscratch[63:0] in [0x00000000:0xFFFFFFFFFFFFFFFF]
```

Comment:
This is broken; it say mscratch is read-only -1, which means that traps and interrupts into mmode can never return